__tests__/app.test.js

```diff
@@ -72,6 +72,32 @@ describe("/api/articles", () => {
                 });
             });
     });
+    describe("?topic=", () => {
+        test("GET:200 should respond with an array of articles filtered by topic", () => {
+            return request(app)
+                .get("/api/articles?topic=mitch")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.articles).toHaveLength(12);
```

Comment:
should actually check the topic property on the return articles.  Suggested adding to this test